init.el

```diff
@@ -9,6 +9,12 @@
 ;;
 ;;; License: GPLv3
 
+;; Mitigation of GNU Emacs 25.2 enriched text remote code execution
+;; see http://seclists.org/oss-sec/2017/q3/422
+(eval-after-load "enriched"
```

Comment:
Should this be wrapped in a conditional to check the current emacs version? [25.3](https://lists.gnu.org/archive/html/emacs-devel/2017-09/msg00211.html) was released recently and patched this vulnerability, so this patch should only be necessary for emacs versions lower than that.